website/content/docs/platform/k8s/vso/sources/vault/auth/index.mdx

```diff
@@ -0,0 +1,501 @@
+kubernetes auth method. The service account must be configured in the Kubernetes namespace of the requesting resource.
+Meaning, if a resource like a `VaultStaticSecret` is created in the `apps` namespace, the service account must be in
+the apps namespace. The rationale behind this approach is to ensure that cross namespace access is not possible.
+
+## Vault authentication globals
+
+The `VaultAuthGlobal` resource is a global configuration that allows you to share a single authentication configuration
+across a set of VaultAuth resources. This is useful when you have multiple VaultAuth resources that share the
+same base configuration. For example, if you have multiple VaultAuth resources that all authenticate to Vault
+using the same auth backend, you can create a single VaultAuthGlobal resource that defines the configuration
+common to all VaultAuth instances. Options like `mount`, `method`, `namespace`, and method specific configuration
+can all be inherited from the VaultAuthGlobal resource. Any field in the VaultAuth resource can be inherited
+from a VaultAuthGlobal instance. Typically, most fields are inherited from the VaultAuthGlobal,
+fields like `role`, and credential provider specific fields like `serviceAccount` are usually set on the referring
+VaultAuth instance, since they are more specific to the application that requires the VaultAuth resource.
+
+*See [VaultAuthGlobal spec][vag-spec]  and [VaultAuth spec][va-spec] for the complete list of available fields.*
+
+
+## VaultAuthGlobal configuration inheritance
+
+- The configuration in the VaultAuth resource takes precedence over the configuration in the VaultAuthGlobal resource.
+- The VaultAuthGlobal can reside in any namespace, but must allow the namespace of the VaultAuth resource to reference it.
+- Default VaultAuthGlobal resources are denoted by the name `default` and are automatically referenced by all VaultAuth resources
+  when `spec.vaultAuthGlobalRef.allowDefault` is set to `true` and VSO is running with the `allow-default-globals`
+  option set in the `-global-vault-auth-options` flag (the default).
+- When a `spec.vaultAuthGlobalRef.namespace` is set, the search for the default VaultAuthGlobal resource is
+  constrained to that namespace. Otherwise, the search order is:
+  1. The default VaultAuthGlobal resource in the referring VaultAuth resource's namespace.
+  2. The default VaultAuthGlobal resource in the Operator's namespace.
+
+
+## Sample use cases and configurations
+
+The following sections provide some sample use cases and configurations for the VaultAuthGlobal resource. These
+examples demonstrate how to use the VaultAuthGlobal resource to share a common authentication configuration across a
+set of VaultAuth resources. Like other namespaced VSO custom resource definitions, there can be many VaultAuthGlobal
+resources configured in a single Kubernetes cluster.
+
+### Multi-tenant application with shared authentication backend
```

Comment:
Dumb question, but what does "multi-tenant" mean in these scenarios?

Reply:
It was meant to convey that each application had it own Vault resources. I have replaced it with "Multiple applications" to avoid any confusion. Updated in 6fedc2452d3436ea46e1351316dede50cff6591f

Reply:
Updated in 6fedc2452d3436ea46e1351316dede50cff6591f